components/p3a/brave_p3a_service.cc

```diff
@@ -56,7 +56,7 @@ constexpr char kP2AServerUrl[] = "https://p2a.brave.com/";
 
 constexpr uint64_t kDefaultUploadIntervalSeconds = 60;  // 1 minute.
 
-// TODO(iefremov): Provide moar histograms!
+// TODO(iefremov): Provide more histograms!
```

Comment:
I know, it's cute. I tend to leave cute alone when it's with more cute, this seems lonely.